README.md

```diff
@@ -49,6 +49,11 @@ An attempt at collecting some of the collections of accessible components but al
 
 - [Responsive Table](http://adrianroselli.com/2017/11/a-responsive-accessible-table.html) - A full tutorial for a responsive, accessible table. [Table demo here](https://codepen.io/aardrian/pen/YEKmxP?editors=1000)
 
+## Tabs
+
+- [A11y Tabs](https://github.com/agnosticui/a11y-tabs) - A lightweight (<1Kb) JavaScript package to facilitate a11y-compliant tabbed interfaces by [Rob Levin](https://developtodesign.com/).
+- [AgnosticUI Tabs](https://www.agnosticui.com/docs/components/tabs.html) - An a11y-compliant tabbed interface component that works in React, Vue 3, Svelte, and Angular.
```

Comment:
@Kolombiken I took the liberty of adding this link too. I didn't have anogisticui.com launched when I originally submitted this. Interestingly enough, the `A11y Tabs` was written as a first attempt when I only had HTML and CSS component and wanted to add in the JavaScript functionality. But, I think it informed an even more interesting component in AgnosticUI. In fact, if you see the tabs used on the front page of agnosticui.com it's in Vue 3 using content projection to accomodate the SVG icons. However, if you tab into and arrow through those (even Home and End key), you'll notice it's a11y compliant. Unfortunately, Vitepress has some a11y issues still which messes up my site's compliance but I've submitted them a ticket for that too. https://github.com/vuejs/vitepress/issues/466

Ok, lot's of info haha. Thanks!

Reply:
Ok! I'm not sure what the word a11y compliant mean. But that could be adjusted later. Merging this now. Thanks!